awx/api/urls/urls.py

```diff
@@ -121,7 +121,7 @@
     re_path(r'^inventories/', include(inventory_urls)),
     re_path(r'^hosts/', include(host_urls)),
     re_path(r'^host_metrics/', include(host_metric_urls)),
-    re_path(r'^host_metric_summary_monthly/$', HostMetricSummaryMonthlyList.as_view(), name='host_metric_summary_monthly_list'),
+    # re_path(r'^host_metric_summary_monthly/$', HostMetricSummaryMonthlyList.as_view(), name='host_metric_summary_monthly_list'), # TBD
```

Comment:
Instead of TBD (in multiple places) can we put a more descriptive comment in like:
```
# This will be used in v1.X of Analytics, commented out for now.
```

That way if someone sees it they have a better understanding of why its commented out?

Reply:
😱when did our bot got this good